packages/config/README.md

```diff
@@ -346,8 +336,8 @@ variables.
 
 _Type_: `NetlifyClient?`
 
-Netlify [JavaScript client instance](https://github.com/netlify/js-client) used to retrieve [`siteInfo`](#siteinfo),
-[`accounts`](#accounts) and [`addons`](#addons).
```

Comment:
I did a [humio search for `path="/api/v1/sites/*/service-instances"`](
https://cloud.us.humio.com/netlify-us-production/search?live=false&query=path%3D%22%2Fapi%2Fv1%2Fsites%2F*%2Fservice-instances%22&start=1h&tz=America%2FToronto) and have a handful of requests with `ua=netlify/js-client`. Are they not coming from this repository? Do you have a sense of where they are coming from?